font-awesome/css/font-awesome.css

```diff
@@ -1,1338 +1,4 @@
 /*!
  *  Font Awesome 4.0.3 by @davegandy - http://fontawesome.io - @fontawesome
```

Comment:
Did you use some tool to compress the original code? That makes it difficult to see what changed. Also updates to newer versions of Font Awesome might be made impossible.